styles/prosilver/template/topicrow_rating_body.html

```diff
@@ -1,8 +1,8 @@
-<!-- IF rating.S_THANKS_TOPIC_REPUT_VIEW and rating.TOPIC_REPUT and not S_IS_BOT and not rating.S_THANKS_TOPIC_REPUT_VIEW_COLUMN -->
+{% if rating.S_THANKS_TOPIC_REPUT_VIEW and rating.TOPIC_REPUT and not S_IS_BOT and not rating.S_THANKS_TOPIC_REPUT_VIEW_COLUMN %}
 	<span style="display: block;">
-	<!-- IF rating.S_THANKS_REPUT_GRAPHIC -->
-	<span style="display: block; float: left; width: {rating.THANKS_REPUT_GRAPHIC_WIDTH}; height: {rating.THANKS_REPUT_HEIGHT}; background: url({rating.THANKS_REPUT_IMAGE_BACK}); background-repeat: repeat-x;"><span style="display: block; height: {rating.THANKS_REPUT_HEIGHT}; width: {rating.TOPIC_REPUT}; background: url({rating.THANKS_REPUT_IMAGE}); background-repeat: repeat-x;"></span></span>&nbsp;
-	<!-- ENDIF -->
-	{L_REPUT}{L_COLON}&nbsp;{rating.TOPIC_REPUT}
+	{% if rating.S_THANKS_REPUT_GRAPHIC %}
+	<span style="display: block; float: left; width: {{ rating.THANKS_REPUT_GRAPHIC_WIDTH }}; height: {{ rating.THANKS_REPUT_HEIGHT }}; background: url({{ rating.THANKS_REPUT_IMAGE_BACK }}); background-repeat: repeat-x;"><span style="display: block; height: {{ rating.THANKS_REPUT_HEIGHT }}; width: {{ rating.TOPIC_REPUT }}; background: url({{ rating.THANKS_REPUT_IMAGE }}); background-repeat: repeat-x;"></span></span>&nbsp;
+	{% endif %}
+	{{ lang('REPUT') }}{{ lang('COLON') }}&nbsp;{{ rating.TOPIC_REPUT }}
```

Comment:
Sorry about so many pull requests, I haven't used github before so I wasn't sure how to group it.  I'll make those changes as you've suggested and test them then recommit.